app/src/main/java/com/mashup/lastgarden/ui/onboarding/TutorialFragment.kt

```diff
@@ -0,0 +1,76 @@
+package com.mashup.lastgarden.ui.onboarding
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.mashup.base.autoCleared
+import com.mashup.base.extensions.loadImage
+import com.mashup.base.image.GlideRequests
+import com.mashup.lastgarden.R
+import com.mashup.lastgarden.databinding.FragmentTutorialBinding
+import com.mashup.lastgarden.ui.BaseViewModelFragment
+import dagger.hilt.android.AndroidEntryPoint
+import javax.inject.Inject
+
+@AndroidEntryPoint
+class TutorialFragment(private val page: Int) : BaseViewModelFragment() {
```

Comment:
Fragment의 생성자를 주입하는 방식 대신 Bundle로 값을 주입하는 방식으로 변경하면 좋을 것 같아요!
이유는 시스템에 의해서 앱이 죽었다가 다시 살아날 때 Fragment 생성자에 넘겼던 값들은 복원은 안해주기 때문이에요 ㅠ

```suggestion
class TutorialFragment() : BaseViewModelFragment() {

    companion object {
           private const val PAGE_TYPE = "page_type"
           fun createIntance(page: Int) = TutorialFragment().apply {
                   arguments = Bundle().apply {
                    putInt(PAGE_TYPE, page)
                }
           }
    }
```